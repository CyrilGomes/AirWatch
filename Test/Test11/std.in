1
government@mail.com
123456
2
1000
4
3
