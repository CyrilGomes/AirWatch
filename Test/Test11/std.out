
=========================
 Welcome to AirWatch :
=========================

1. Login
2. Register
3. Exit
> :

Login
-----

Email :
Password :

=========================
 Main menu :
=========================

1. Sensor analytics
2. Cleaner analytics
3. Register Air Cleaning Company
4. Log out
> :

Cleaner Contribution
--------------------

Cleaner ID :

=========================
 Main menu :
=========================

1. Sensor analytics
2. Cleaner analytics
3. Register Air Cleaning Company
4. Log out
> :

=========================
 Welcome to AirWatch :
=========================

1. Login
2. Register
3. Exit
> : 
